crates/sui-adapter-transactional-tests/tests/derived_objects/claim_twice@v2.snap

```diff
@@ -0,0 +1,30 @@
+---
+source: external-crates/move/crates/move-transactional-test-runner/src/framework.rs
+---
+processed 5 tasks
+
+init:
+A: object(0,0)
+
+task 1, lines 9-31:
+//# publish
+created: object(1,0)
+mutated: object(0,1)
+gas summary: computation_cost: 1000000, storage_cost: 6460000,  storage_rebate: 0, non_refundable_storage_fee: 0
+
+task 2, line 33:
+//# run a::m::create_parent --sender A
+created: object(2,0)
+mutated: object(0,0)
+gas summary: computation_cost: 1000000, storage_cost: 2234400,  storage_rebate: 0, non_refundable_storage_fee: 0
+
+task 3, line 35:
+//# run a::m::derive_object --sender A --args object(2,0) 0
+created: object(3,0), object(3,1)
+mutated: object(0,0), object(2,0)
+gas summary: computation_cost: 1000000, storage_cost: 5981200,  storage_rebate: 2212056, non_refundable_storage_fee: 22344
+
+task 4, line 37:
+//# run a::m::derive_object --sender A --args object(2,0) 0
+Error: Transaction Effects Status: Move Runtime Abort. Location: sui::derived_object::claim (function index 0) at offset 20, Abort Code: 13835058231375822849
```

Comment:
Something to think about @tzakian... since this code is going to be a bit annoying when it appears in snapshots

Reply:
We should probably render these shouldn't we, or at least render the `code =` code of the error.